wgsl/index.bs

```diff
@@ -1872,14 +1885,19 @@ Note: When no conversion is performed, the conversion rank is zero.
       <td>There are no automatic conversions between other types.
 </table>
 
-The type `T` is the <dfn noexport>concretization</dfn> of type `S` if:
+For a given expression `e: S`, the type `T` is the
```

Comment:
The original text describes a mapping from type to types. 
The next paragraph describes the concretization of a value.  There's no need to make this paragraph about values directly.

Reply:
The way it read to me is that there is a single concretization of a type. This tries to clarify that the concretization occurs per expression.

Reply:
I think our disagreement here is actually elsewhere.

There are two uses of "concretization"
- where it's used to define effective-value-type, and the unique concretization of a value is the right concept. It's well worded.
- in step 2 of overload resolution, in the sub-bullet:
> That is, overload resolution [will](https://gpuweb.github.io/gpuweb/wgsl/#behavioral-requirements) [concretize](https://gpuweb.github.io/gpuweb/wgsl/#concretization) the type of an expression unless all its subexpressions are [const-expressions](https://gpuweb.github.io/gpuweb/wgsl/#const-expressions).

That's a bit vague in the sense that the result type could be one of many concrete types, depending on what overloads are available.  This really is a Note, and could be worded better.

That bullet could say instead

> Note: As a consequence, if any subexpression in the phrase is not a const-expression, then all subexpressions in the phrase must have concrete type.

?